en/docs/how-tos/rate-limiting/distributed-throttling.md

```diff
@@ -186,7 +192,7 @@
 
 ### Conditional throttling
 
```

Comment:
Shall we point to APIM doc here to better understand on how to add the throttle policies from admin portal

Reply:
It is already included in https://mg.docs.wso2.com/en/latest/how-tos/rate-limiting/distributed-throttling/#configure-and-enable-conditional-throttling section